oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/analysis/metrics/Event.java

```diff
@@ -100,7 +100,7 @@ protected StorageID id0() {
     @Column(name = TYPE)
     private String type;
 
-    @Column(name = MESSAGE)
+    @Column(name = MESSAGE, length = 2000)
```

Comment:
```suggestion
    @Column(name = MESSAGE, storageOnly = true, length = 2000)
```

@kezhenxu94 Please confirm whether we should do this. Indexing a length=2000 string doesn't make sense to me. If this needs to be indexing, we need `MatchQuery` for Elasticsearch.